app/src/main/java/com/duckduckgo/app/browser/defaultbrowsing/prompts/DefaultBrowserPromptsExperimentImpl.kt

```diff
@@ -154,9 +165,10 @@ class DefaultBrowserPromptsExperimentImpl @Inject constructor(
     }
 
     private suspend fun evaluate() {
+        val isOnboardingComplete = userStageStore.getUserAppStage() == AppStage.ESTABLISHED
```

Comment:
Has @nalcalag looked at this? In the past we’ve had issues understanding if onboarding was complete in some edge cases, so just want to double check.

Reply:
Thanks for the heads-up @malmstein . This check is correct, when onboarding is completed, the `UserAppStage` should be set to `ESTABLISHED`. This means all onboarding dialogs, both in-context and on the new tab page, will have already been shown.

There are other methods to determine if parts of the onboarding process have been completed, such as checking for dialogs on the new tab page, etc. I’m not sure about the specific requirements for this feature, but feel free to reach out to me if needed @LukasPaczos .